sunpy/io/tests/test_fits.py

```diff
@@ -176,3 +177,21 @@ def test_warn_longkey():
                                'goodkey': 'test'})
     assert 'GOODKEY' in fits.keys()
     assert 'BADLONGKEY' not in fits.keys()
+
+
+def test_read_memmap():
+    # Check that the FITS reader can read a memmap without memmap argument by default
+    data, header = sunpy.io._fits.read(TEST_AIA_IMAGE)[0]
+    if data.base is None:
+        assert False
```

Comment:
Do we need this?

Reply:
Oh, we don't need that, whenever the `data.base` is `None` the test will fail.